nixos/modules/installer/netboot/netboot.nix

```diff
@@ -18,17 +18,15 @@ with lib;
 
   };
 
-  config = {
-
-    boot.loader.grub.version = 2;
-
+  config = rec {
```

Comment:
I'm going to restore the grub version for not-aarch64-linux.

Reply:
Shouldn't it also be '2' for aarch64?

Reply:
Turns out we don't need this, anyway. Defaults to 2.